### PR TITLE
pofile page with placeholder and dummy data

### LINKED ISSUE #13: Profile Page

### PR DESCRIPTION
**Pull request by HowardYaw**
resolve #13

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import * as React from "react";
 import { NavigationContainer } from "@react-navigation/native";
 import { createStackNavigator } from "@react-navigation/stack";
-import { HomeScreen, SignUpScreen } from "./pages";
+import { HomeScreen, SignUpScreen, ProfileScreen } from "./pages";
+// import { ProfileScreen } from "./pages/Profile/Profile";
```
Comment (LUXIANZE) [inline src/App.js]: remove this line

File: src/App.js [hunk 2]
```diff
@@ -11,6 +12,7 @@ const App = () => {
       <Stack.Navigator>
         <Stack.Screen name="Home" component={HomeScreen} />
         <Stack.Screen name="SignUp" component={SignUpScreen} />
+        <Stack.Screen name="Profile" component={ProfileScreen} />
```
Comment (LUXIANZE) [inline src/App.js]: After merging with #23 add page header title to it

**Review by LUXIANZE — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (HowardYaw) [reply src/App.js]: solve in d64f27a4351abcc9d759dbb9d55e3b8f238d6006

File: src/App.js (hunk 2 above)
Comment (HowardYaw) [reply src/App.js]: solve in d64f27a4351abcc9d759dbb9d55e3b8f238d6006

**Review by LUXIANZE — APPROVED** [review]

**Pull request merged**